### PR TITLE
Refactor small sections of code and tests

### PR DESCRIPTION
**Pull request by kishen-v**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
This PR contains changes that clean up certain function definitions.

**Release note**:
```
none
```

### CODE REVIEW

File: tests/e2e/README.md [hunk 1]
```diff
@@ -2,16 +2,23 @@
...
+Tests marked with `[labels]` requires specific labels to be set for each node of the cluster, if not set these tests will be skipped.
+
+```
+kubectl label nodes <node-name> powervs.kubernetes.io/cloud-instance-id=<GUID of the workspace>
+kubectl label nodes <node-name> powervs.kubernetes.io/pvm-instance-id=<Instance ID of the PowerVS instance>
 ```
-kubectl label nodes multinode-kube-master powervs.kubernetes.io/cloud-instance-id=7845d372-d4e1-46b8-91fc-41051c984601
-kubectl label nodes multinode-kube-master powervs.kubernetes.io/pvm-instance-id=638667f8-a4d3-46d0-9fa6-ddc621100407
+
+The ProviderID of the format(v2) - ibmpowervs://<region>/<zone>/<service_instance_id>/<powervs_machine_id> needs to be set on the nodes of the cluster. This can be achieved by deploying clusters through [CAPIBM](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud), or be set by editing the node configuration using the `kubectl edit node <node-name>` command and inserting the ProviderID, for example:
```
Comment (yussufsh) [inline tests/e2e/README.md:18]:
```
Set the ProviderID on the cluster nodes as: `ibmpowervs://<region>/<zone>/<service_instance_id>/<powervs_machine_id>`, for example:
```sh
spec:
  providerID: ibmpowervs://syd/syd05/862032d5-xxxx-xxxx-xxxx-c18594456427/2c6cbaec-xxxx-xxxx-xxxx-a6aa35315596
```

**Review by yussufsh — CHANGES_REQUESTED** [review]
@kishen-v please rebase. Thanks.

**Comment by kishen-v** [comment]
Hi @yussufsh,

Rebased the PR with main and addressed the review comment.
Thanks!

**Review by yussufsh — APPROVED** [review]
/lgtm

**Pull request merged**